sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -392,24 +406,35 @@ def test_check_estimator():
     assert_raises_regex(AssertionError, msg, check_estimator,
                         LargeSparseNotSupportedClassifier)
 
-    # non-regression test for estimators transforming to sparse data
-    check_estimator(SparseTransformer())
-
-    # doesn't error on actual estimator
-    check_estimator(AdaBoostClassifier)
-    check_estimator(AdaBoostClassifier())
-    check_estimator(MultiTaskElasticNet)
-    check_estimator(MultiTaskElasticNet())
-
-    # doesn't error on binary_only tagged estimator
-    check_estimator(TaggedBinaryClassifier)
-
     # does error on binary_only untagged estimator
     msg = 'Only 2 classes are supported'
     assert_raises_regex(ValueError, msg, check_estimator,
                         UntaggedBinaryClassifier)
 
 
+@pytest.mark.parametrize(
+    "estimator",
+    [
+        # non-regression test for estimators transforming to sparse data
+        SparseTransformer(),
+
+        # doesn't error on actual estimator
+        LogisticRegression,
+        LogisticRegression(),
```

Comment:
Previously this run on `AdaBoostClassifier` which took significant time (around 3 sec) for two checks. Since it looks like this only intends to check that initialized/non initialized estimators work, replaced it by a faster estimator.

`AdaBoostClassifier` is picked up by common tests anyway.